GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -253,6 +255,13 @@ public FormBrowse(GitUICommands commands, BrowseArguments args)
         {
             SystemEvents.SessionEnding += (sender, args) => SaveApplicationSettings();
 
+            _isFileBlameHistory = args.IsFileBlameHistory;
+            if (_isFileBlameHistory is not null && _isFileBlameHistory.Value)
```

Comment:
If the `bool?` is kept instead of an `enum`:
```suggestion
            if (_isFileBlameHistory is true)
```

Reply:
ok